snippets/auth.go

```diff
@@ -185,6 +185,35 @@ func getUserByPhone(ctx context.Context, client *auth.Client) *auth.UserRecord {
 	return u
 }
 
+func bulkGetUsers(ctx context.Context, client *auth.Client) {
+	uid := "uid1"
+	email := "user@example.com"
+	phone := "+15555551234"
+	providerID := "google.com"
+	providerUID := "google_uid1"
+	// [START bulk_get_users_golang]
+	getUsersResult, err := client.GetUsers(ctx, []auth.UserIdentifier{
+		auth.UIDIdentifier{UID: uid},
```

Comment:
May be inline the values of uid, email etc for clarity.

Reply:
Done. I've also applied this to bulk delete (despite it just being uids.)